cirq-google/cirq_google/engine/engine.py

```diff
@@ -195,7 +203,7 @@ def run(
         param_resolver: cirq.ParamResolver = cirq.ParamResolver({}),
         repetitions: int = 1,
         processor_ids: Sequence[str] = ('xmonsim',),
-        gate_set: Optional[Serializer] = None,
+        gate_set: Serializer = None,
```

Comment:
If it's not too much hassle, could you add deprecation notices to these fields? In my discussions I haven't heard arguments against removing this field in the future.

Reply:
Could do that here, of in a separate PR; which do you prefer? I think there are probably still some usages of this parameter in tests and example notebooks that we'd want to change, so I might prefer making a separate PR to track all that down.

Reply:
Separate PR sounds good!